wetterdienst/dwd/radar/metadata.py

```diff
@@ -0,0 +1,127 @@
+from enum import Enum
+
+
+class RadarDate(Enum):
+    """
+    Enumeration for pointing to different radar dates.
+    """
+
+    LATEST = "latest"
+    CURRENT = "current"
+    MOST_RECENT = "most_recent"
+
+
+class RadarDataFormat(Enum):
+    """
+    Radar data formats.
+    """
+
+    BINARY = "binary"
+    BUFR = "bufr"
+    HDF5 = "hdf5"
+
+
+class RadarDataSubset(Enum):
+    """
+    HDF5 subset types for radar sweep data.
+
+    https://opendata.dwd.de/weather/radar/sites/sweep_pcp_v/boo/hdf5/
+    """
+
+    SIMPLE = "simple"
+    POLARIMETRIC = "polarimetric"
+
+
+class RadarParameter(Enum):
+    """
+    All available radar moments.
+    """
+
+    # /composites
+    # https://docs.wradlib.org/en/stable/notebooks/fileio/wradlib_radar_formats.html#German-Weather-Service:-RADOLAN-(quantitative)-composit  # noqa:E501,B950
+
+    # https://opendata.dwd.de/weather/radar/composit/
+    # TODO: "FX" was missing here. OTOH, "pp" doesn't
+    #       seem to be present on opendata.dwd.de.
+    FX_REFLECTIVITY = "fx"
+    PG_REFLECTIVITY = "pg"
+    # PP_REFLECTIVITY = "pp"
+    WX_REFLECTIVITY = "wx"
+    WN_REFLECTIVITY = "wn"
+    RX_REFLECTIVITY = "rx"
+
+    # /radolan
+    # https://opendata.dwd.de/weather/radar/radolan/
+    RW_REFLECTIVITY = "rw"
+    RY_REFLECTIVITY = "ry"
+    SF_REFLECTIVITY = "sf"
+
+    # /radvor
+    # https://opendata.dwd.de/weather/radar/radvor/
+    RE_REFLECTIVITY = "re"
+    RQ_REFLECTIVITY = "rq"
```

Comment:
Dear @kmuehlbauer and @meteoDaniel,

I've also added support for acquisition of "radvor" data from [1].

I see that the data there has a suffix of `_000`-`_120`(in steps of 5) for "re" data [2] and `_000`, `_060`, `_120` for "rq" data [3]. Is this something we should account for when acquiring data?

Also, do you believe `RE_REFLECTIVITY` and `RQ_REFLECTIVITY` are appropriate enum names here?

With kind regards,
Andreas.

[1] https://opendata.dwd.de/weather/radar/radvor/
[2] https://opendata.dwd.de/weather/radar/radvor/re/
[3] https://opendata.dwd.de/weather/radar/radvor/rq/

Reply:
RADVOR is a forecast product and the steps you are mention are the lead time. I do not have any requirements to such a data. Nice feature.

Reply:
@amotl This might be a nice feature for users wanting kind of forecast. I've not used it yet and can't say much about it.

Reply:
All right. Then just leave it like it is. `RE_REFLECTIVITY` and `RQ_REFLECTIVITY` are not obviously wrong names?